cernopendata/modules/fixtures/data/records/cms-trigger-information-2010.json

```diff
@@ -0,0 +1,28 @@
CMSSW_3_8_1_onlpatch5</a></td><td><a href=\"/record/6140\">/cdaq/physics/Run2010/v7.1/HLT/V1</a></td></tr>\n      <tr><td>146944 - 147116</td><td><a href=\"https://github.com/cms-sw/cmssw-cvs/tree/CMSSW_3_8_1_onlpatch5\">CMSSW_3_8_1_onlpatch5</a></td><td><a href=\"/record/6141\">/cdaq/physics/Run2010/v7.1/HLT/V2</a></td></tr>\n      <tr><td>147196</td><td><a href=\"https://github.com/cms-sw/cmssw-cvs/tree/CMSSW_3_8_1_onlpatch6\">CMSSW_3_8_1_onlpatch6</a></td><td><a href=\"/record/6142\">/cdaq/physics/Run2010/v8.0/HLT/V2</a></td></tr>\n      <tr><td>147206 - 147284</td><td><a href=\"https://github.com/cms-sw/cmssw-cvs/tree/CMSSW_3_8_1_onlpatch6\">CMSSW_3_8_1_onlpatch6</a></td><td><a href=\"/record/6143\">/cdaq/physics/Run2010/v8.0/HLT/V3</a></td></tr>\n      <tr><td>147390 - 147454</td><td><a href=\"https://github.com/cms-sw/cmssw-cvs/tree/CMSSW_3_8_1_onlpatch6\">CMSSW_3_8_1_onlpatch6</a></td><td><a href=\"/record/6144\">/cdaq/physics/Run2010/v8.1/HLT/V1</a></td></tr>\n    <tr><td>147749 - 147757</td><td><a href=\"https://github.com/cms-sw/cmssw-cvs/tree/CMSSW_3_8_1_onlpatch7\">CMSSW_3_8_1_onlpatch7</a></td><td><a href=\"/record/6145\">/cdaq/physics/Run2010/v8.1/HLT/V2</a></td></tr>\n      <tr><td>147926 - 148057</td><td><a href=\"https://github.com/cms-sw/cmssw-cvs/tree/CMSSW_3_8_1_onlpatch7\">CMSSW_3_8_1_onlpatch7</a></td><td><a href=\"/record/6145\">/cdaq/physics/Run2010/v8.2/HLT/V1</a></td></tr>\n      <tr><td>148029 - 148058</td><td><a href=\"https://github.com/cms-sw/cmssw-cvs/tree/CMSSW_3_8_1_onlpatch7\">CMSSW_3_8_1_onlpatch7</a></td><td><a href=\"/record/6145\">/cdaq/physics/Run2010/v8.2/HLT/V2</a></td></tr>\n      <tr><td>148819 - 148829</td><td><a href=\"https://github.com/cms-sw/cmssw-cvs/tree/CMSSW_3_8_5_onlpatch3\">CMSSW_3_8_5_onlpatch3</a></td><td><a href=\"/record/6145\">/cdaq/physics/Run2010/v9.1/HLT/V1</a></td></tr>\n      <tr><td>148828</td><td><a href=\"https://github.com/cms-sw/cmssw-cvs/tree/CMSSW_3_8_5_onlpatch3\">CMSSW_3_8_5_onlpatch3</a></td><td><a href=\"/record/6145\">/cdaq/physics/Run2010/v9.2/HLT/V1</a></td></tr>\n      <tr><td>148859 - 148864</td><td><a href=\"https://github.com/cms-sw/cmssw-cvs/tree/CMSSW_3_8_5_onlpatch3\">CMSSW_3_8_5_onlpatch3</a></td><td><a href=\"/record/6145\">/cdaq/physics/Run2010/v9.2/HLT/V2</a></td></tr>\n      <tr><td>148952 - 149063</td><td><a href=\"https://github.com/cms-sw/cmssw-cvs/tree/CMSSW_3_8_5_onlpatch3\">CMSSW_3_8_5_onlpatch3</a></td><td><a href=\"/record/6145\">/cdaq/physics/Run2010/v9.3/HLT/V1</a></td></tr>\n      <tr><td>149181 - 149182</td><td><a href=\"https://github.com/cms-sw/cmssw-cvs/tree/CMSSW_3_8_5_onlpatch3\">CMSSW_3_8_5_onlpatch3</a></td><td><a href=\"/record/6145\">/cdaq/physics/Run2010/v9.4/HLT/V1</a></td></tr>\n      <tr><td>149291 - 149442</td><td><a href=\"https://github.com/cms-sw/cmssw-cvs/tree/CMSSW_3_8_5_onlpatch3\">CMSSW_3_8_5_onlpatch3</a></td><td><a href=\"/record/6146\">/cdaq/physics/Run2010/v9.5/HLT/V1</a></td></tr>\n      </tbody>\n      </table>\n      "
```

Comment:
The table looks good, but record IDs used for config files such as `/cdaq/physics/firstCollisions10/v2.0/HLT_7TeV/V5` are wrong and would need to be updated...

Reply:
... I'll push my updates into this branch with the recid, year and run period formatting and we can amend configuration files as the next step.